src/lib/utils/thorchain/hooks/useSendThorTx.tsx

```diff
@@ -270,10 +270,8 @@ export const useSendThorTx = ({
     if (!accountId) return
     if (!transactionType) return
     if (!estimateFeesArgs) return
-    if (isToken(asset.assetId) && !inboundAddressData) return
-
-    await checkLedgerAppOpenIfLedgerConnected(asset.chainId)
     if (accountNumber === undefined) return
+    if (isToken(asset.assetId) && !inboundAddressData) return
```

Comment:
diff is drunk, `accountNumber` has been brought back up, and `checkLedgerAppOpenIfLedgerConnected()` back down.

Reply:
seeing this after commenting on the previous pr. that answers that.

Reply:
And diff looks good now 🎉